modules/siddhi-extensions/event-table/pom.xml

```diff
@@ -76,23 +95,40 @@
 
     </dependencies>
 
+    <properties>
+        <org.eclipse.osgi.version>3.7.0.v20110613</org.eclipse.osgi.version>
+        <version.equinox.osgi.services>3.3.100.v20120522-1822</version.equinox.osgi.services>
+        <hazelcast.version>3.5.0.wso2v1</hazelcast.version>
+    </properties>
+
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
```

Comment:
@grainier 
Have we defined the scr-plugin anywhere in the parent? If not, you need to provide all the configuration details here properly. i.e. the version, goals need to be specified. You can specify them in pluginManagement section and then just declare the group id and artifact id in the plugin section.